run_create_cloudwatch_alarm.py

```diff
@@ -27,40 +27,17 @@ def run_create_cloudwatch_alarm_lambda(name, settings):
         print('sns topic: "%s" is not exists in %s' % (settings['SNS_TOPIC_NAME'], alarm_region))
         raise Exception()
 
-    dimensions = list()
-    dimensions.append({"Name": "FunctionName", "Value": name})
-    dimensions.append({"Name": "Resource", "Value": name})
-
     metrics = list()
-    dd = dict()
-    dd['Id'] = 'availability'
-    dd['Expression'] = '100 - 100 * errors / invocations'
-    dd['Label'] = 'Success rate (%)'
-    dd['ReturnData'] = True
-    metrics.append(dd)
-
     dd = dict()
     dd['Id'] = 'errors'
     dd['MetricStat'] = dict()
     dd['MetricStat']['Metric'] = dict()
-    dd['MetricStat']['Metric']['Dimensions'] = dimensions
+    dd['MetricStat']['Metric']['Dimensions'] = []
```

Comment:
```suggestion
    dd['MetricStat']['Metric']['Dimensions'] = list()
```
list() 선언으로 변경하는게 어떨까요?

Reply:
@singetta // 제안 감사합니다. 저도 놓치고 있었네요 ;)